gpt_computer_agent/gpt_computer_agent.py

```
@@ -1,10 +1,10 @@
 try:
-    from .agent.chat_history import *
-    from .agent.assistant import *
+    from .assistant.chat_history import *
```

Ruff (0.8.2)

`from .assistant.chat_history import *` used; unable to detect undefined names

(F403)